tests/containers/secret.pm

```diff
@@ -37,7 +41,6 @@ sub run {
 
     # "podman secret exists" was added to podman 4.5.0 according to
     # https://github.com/containers/podman/blob/main/RELEASE_NOTES.md#450
-    my $podman_version = get_podman_version();
     if (version->parse($podman_version) >= version->parse('4.5.0')) {
```

Comment:
If we're returning early, we can get rid of this condition.

Reply:
> If we're returning early, we can get rid of this condition.

No because podman secret works just fine on 12-SP3.

Reply:
Wasn't the discussion about `Unrecognized command podman secret exists`?

Reply:
The original issue was indeed that, this seems to another issue, nevermind.

Reply:
> If we're returning early, we can get rid of this condition.

No because the earlier test is for an earlier version of podman.